src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped work!");
```

Comment:
The message should be informative and indicate the type of `Machine` is working right now. Consider changing the message to match the format used in the `doWork` method, such as 'Bulldozer stopped working!'.